### DISCUSSION TITLE
Adding New Device WPK

### DISCUSSION THREAD

**Discussion by NTV20244** [Koenkk/zigbee2mqtt#22965] [Devices]
Hello, i prepared code for unsupported new curtain system, please upload the database.
Thanks.

![wpk](https://github.com/Koenkk/zigbee2mqtt/assets/169887598/7d593db3-c204-4569-88a8-4d587164885a)

```
const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
const legacy = require('zigbee-herdsman-converters/lib/legacy');
const tz = require('zigbee-herdsman-converters/converters/toZigbee');
const exposes = require('zigbee-herdsman-converters/lib/exposes');
const reporting = require('zigbee-herdsman-converters/lib/reporting');
const modernExtend = require('zigbee-herdsman-converters/lib/modernExtend');
const e = exposes.presets;
const ea = exposes.access;

const definition = {
    fingerprint: [
        {modelID: 'TS0601', manufacturerName: '_TZE204_lh3arisb'},
    ],
    model: 'WPK',
    vendor: 'Novato',
    description: 'Smart Curtain Track',
        fromZigbee: [legacy.fromZigbee.tuya_cover, fz.ignore_basic_report],
        toZigbee: [legacy.toZigbee.tuya_cover_control],
        exposes: [e.cover_position().setAccess('position', ea.STATE_SET)],

    }
    module.exports = definition;
```

**Comment by Koenkk** [comment]
Added!

Could you also submit a picture for the docs? This can be done by clicking [here](https://github.com/Koenkk/zigbee2mqtt.io/upload/master/public/images/devices). 

Make sure that:
- The filename is `MODEL.png` (update `model` accordingly) 
- The size is 512x512
- The background is transparent (use e.g. [Adobe remove background](https://new.express.adobe.com/tools/remove-background))

**Reply by NTV20244** [reply]
Thank you for adding but cant sumbit picture.
```
Uploads are disabled.
File uploads require push access to this repository.
```

**Reply by Koenkk** [reply]
Does it work when clicking -> Add file -> Create new file [here](https://github.com/Koenkk/zigbee2mqtt.io/tree/master/public/images/devices)?
